test/integration_test/kubevirt_hyperconv_test.go

```diff
@@ -308,6 +308,11 @@ func kubeVirtSimulateOCPUpgrade(t *testing.T) {
 		log.Infof("\nStart OCP upgrade simulation on node: %s", nodeName)
 		testStatesNode := getTestStatesForNode(t, ctxs, nodeName, allNodes)
 
+		if len(testStatesNode) == 0 {
+			log.Infof("No VMs are present on node: %s. Skipping restart of PX on this node", nodeName)
```

Comment:
nit: Skipping restart... -> Skipping node 

since we are skipping the entire the test for this node

Reply:
Fixed.